Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM resin/armv7hf-debian:jessie
+
+RUN apt-get update \
+	&& apt-get install -y axel wget htop nano libjpeg-dev libconfig++-dev\
+	libevent-dev git freebsd-glue curl python bison flex\
+	&& apt-get clean \
+        && apt-get autoremove -qqy 
+
+RUN curl -sL https://deb.nodesource.com/setup | bash -
+RUN apt-get install -y build-essential nodejs
+
+RUN mkdir -p /opt/adapteva && cd /opt/adapteva \
+	&& axel -n 10 http://ftp.parallella.org/esdk/beta/esdk.2014.11.20150522_linux_armv7l.tar.gz && tar -xf esdk.2014.11.20150522_linux_armv7l.tar.gz && rm esdk.2014.11.20150522_linux_armv7l.tar.gz \
+	&& ln -sTf /opt/adapteva/esdk.2014.11 /opt/adapteva/esdk
+
+ENV EPIPHANY_HOME /opt/adapteva/esdk
```

Comment:
I'd move that before the above `curl` and use the variable in the command
```Dockerfile
RUN mkdir -p $EPIPHANY_HOME \
    && curl -sL http://ftp.parallella.org/esdk/beta/esdk.2014.11.20150522_linux_armv7l.tar.gz | tar xz -C $EPIPHANY_HOME --strip-components=1
```

Also, a ESDK_VERSION env var would make sense
```Dockerfile
ENV EPIPHANY_HOME /opt/adapteva/esdk
ENV ESDK_VERSION 2014.11.20150522

RUN mkdir -p $EPIPHANY_HOME \
    && curl -sL http://ftp.parallella.org/esdk/beta/esdk.$ESDK_VERSION_linux_armv7l.tar.gz | tar xz -C $EPIPHANY_HOME --strip-components=1
```